src/items/itemsTable.component.tsx

```diff
@@ -756,15 +739,64 @@ export function ItemsTable(props: ItemTableProps) {
       : undefined,
   });
 
-  React.useEffect(() => {
-    if (isSparesDefinitionDefined)
-      table.setShowAlertBanner(
-        searchParams.get('state') === encodedSparesFilter
-      );
-  }, [encodedSparesFilter, isSparesDefinitionDefined, searchParams, table]);
-
   return (
     <div style={{ width: '100%' }}>
+      {isSparesDefinitionDefined && isSparesFilterApplied && (
+        <Alert
+          color="info"
+          icon={false}
+          sx={() => ({
+            '& .MuiAlert-message': {
+              width: '100%',
+            },
+            borderRadius: 0,
+            fontSize: '1rem',
+            left: 0,
+            p: 0,
+            position: 'relative',
+            right: 0,
+            top: 0,
+            width: '100%',
+            zIndex: 2,
+          })}
+        >
+          <Grid container alignItems="center" sx={{ px: 1, py: 0.5 }}>
+            <Grid size={2}> </Grid>
```

Comment:
```suggestion
            <Grid size={2} />
```
Same comment as before, I suspect it should fix itself when merged though.